modules/nw-ptp-configuring-linuxptp-services-as-boundary-clock.adoc

```diff
@@ -29,7 +29,7 @@ spec:
   profile: <2>
   - name: "profile1" <3>
     interface: "" <4>
```

Comment:
The interface should not be specified, although it's empty. I guess It is better to leave it  out?